src/python/pants/backend/jvm/tasks/jvm_compile/zinc/zinc_compile.py

```diff
@@ -332,6 +332,9 @@ def create_extra_products_for_targets(self, targets):
                     )
 
     def post_compile_extra_resources(self, compile_context):
```

Comment:
This method isn't really intended for sideeffects... ideally rather than putting this here it would be closer to the other metrics we report, here: https://github.com/pantsbuild/pants/blob/ad8229e4b17656d8bb1f906b4408f51214a874b5/src/python/pants/backend/jvm/tasks/jvm_compile/jvm_compile.py#L1231-L1238

Reply:
Thanks Stu! This was super helpful to figure out why my change didn't behave as expected!